test/built-ins/RegExp/prototype/sticky/this-regexp.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    `sticky` accessor function invoked on a RegExp instance
+es6id: 21.2.5.12
+info: >
+    21.2.5.12 get RegExp.prototype.sticky
+
+    4. Let flags be the value of R’s [[OriginalFlags]] internal slot.
+    5. If flags contains the code unit "y", return true.
+    6. Return false.
+---*/
+
+assert.sameValue(/./.sticky, false);
+assert.sameValue(/./i.sticky, false);
+assert.sameValue(/./g.sticky, false);
+assert.sameValue(/./gi.sticky, false);
+
+assert.sameValue(/./y.sticky, true);
+assert.sameValue(/./iy.sticky, true);
+assert.sameValue(/./yg.sticky, true);
+assert.sameValue(/./iyg.sticky, true);
```

Comment:
This would improve coverage, so it's hard to say "no", but the same would go for all the RegExp tests (and all the tests for prototype methods of primitive types). I don't think it makes sense to duplicate every assertion (one for "X when value created via literal expression" and one for "X when value created via constructor"), nor do I think that is what you are suggesting. There are probably cases where the construction approach might have some bearing on the behavior under test, but I think those should be clear before testing both forms (otherwise there will always be ambiguity as to whether such duplication is preferable).